rpm/SPECS/openresty-pcre.spec

```diff
@@ -0,0 +1,62 @@
+Name:          openresty-pcre
+Version:       8.39
+Release:       1%{?dist}
+Summary:       Perl-compatible regular expression library for OpenResty
+
+Group:         System Environment/Libraries
+
+License:       BSD
+URL:           http://www.pcre.org/
+Source0:       ftp://ftp.csx.cam.ac.uk/pub/software/programming/pcre/pcre-%{version}.tar.bz2
+
+BuildRequires: libtool
+
+%define pcre_prefix /usr/local/openresty/pcre
+
+# Do not check for provides, our internals are not for others.
+AutoReqProv:   no
+
+%description
+Perl-compatible regular expression library for use by OpenResty ONLY
+
+%package devel
+Summary:       Development files for %{name}
+Group:         Development/Libraries
+Requires:      %{name} = %{version}-%{release}
+
+%description devel
+Development files for Perl-compatible regular expression library for use by OpenResty ONLY
+
+%prep
+%setup -q -n pcre-%{version}
+
+%build
+LDFLAGS="-Wl,-rpath,%{pcre_prefix}/%{_lib}" ./configure \
+  --prefix=%{pcre_prefix} \
+  --libdir=%{pcre_prefix}/%{_lib} \
```

Comment:
We should not use `lib64` on x86_64, for example. Just use the default `lib` directory name should be sufficient.